src/Clide.Interfaces/Commands/ICommandManager.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Clide
+{
+	public interface ICommandManager
```

Comment:
Since we have `ICommandBus` and `ICommand<T>/ICommandHandler<T>` from Merq, I wonder if this naming isn't confusing :(. These commands are UI commands really. Maybe we should switch to `IUICommand`? `IVisualCommand`?